### PR TITLE
KCL V2.3.3 release notes.

### PR DESCRIPTION
**Pull request by yatins47**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # Changelog
+### Latest Release 2.3.3 (December 23, 2020)
+[Milestone#55](https://github.com/awslabs/amazon-kinesis-client/milestone/55)
+* Fixing bug in PrefetchRecordsPublisher which was causing retry storms if initial request fails.
+* Fixing bug where idleTimeBetweenReadsInMillis property was ignored in PollingConfig.
 
 ### Latest Release 2.3.2 (November 19, 2020)
```
Comment (isurues) [inline CHANGELOG.md:7]: Remove Latest here?

File: README.md [hunk 2]
```diff
@@ -63,6 +63,11 @@ The recommended way to use the KCL for Java is to consume it from Maven.
 
 ## Release Notes
 
+### Latest Release 2.3.3 (December 23, 2020)
+[Milestone#55](https://github.com/awslabs/amazon-kinesis-client/milestone/55)
+* Fixing bug in PrefetchRecordsPublisher which was causing retry storms if initial request fails.
+* Fixing bug where idleTimeBetweenReadsInMillis property was ignored in PollingConfig.
+
 ### Latest Release 2.3.2 (November 19, 2020)
```
Comment (isurues) [inline README.md:71]: Same here.

**Review by ashwing — APPROVED** [review]
lgtm

**Review by isurues — APPROVED** [review]
Looks good to me.

**Pull request merged**